src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -3684,6 +3684,11 @@ class Organization: ADOSVTBase
                 # Checking if everyone in the organization has create permission on feed
                 $restrictedGroups = @($roleAssignments | Where-Object {"Project Collection Valid Users" -contains $_.split('\')[-1] })
                 $restrictedGroupsCount = $restrictedGroups.Count
+                if ($this.ControlFixBackupRequired)
+                {           
+                    $backup = @{"Descriptor"= $responseObj.identityDescriptor;"Id"=$responseObj.identityId};             
```

Comment:
Automated fix is not working in scenarios where multiple users along with PCVU are given the permissions